servlet/theLearningServlet/src/main/java/jakartaee/examples/servlet/thelearningservlet/TheLearningServlet.java

```diff
@@ -0,0 +1,270 @@
+ *
+ */
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.logging.Logger;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet implementation class TheLearningServlet
+ *
+ * The @WebServlet annotation defines this class as a servlet. The description
+ * attribute is informative and this will only appear in the app server's
+ * console. The default name of the servlet is the name of the class but with
+ * the first letter made lowercase. In this class it would be
+ * 'theLearningServlet'. You can change this, as I have, in the attribute
+ * urlPatterns. You can have more than one name for a servlet. This is used when
+ * the name used to access the servlet can act as a selector to determine what
+ * is expected of the servlet as it can discover what name was used to call it.
+ *
+ */
+@WebServlet(urlPatterns = "/learning")
+public class TheLearningServlet extends HttpServlet {
+
+    // All methods log their name just to show the order of calling
+    private static final Logger LOG = Logger.getLogger(TheLearningServlet.class.getName());
+
+    /**
+     * Constructor
+     *
+     * Can only be used to initialize class variables or perform tasks before
+     * the servlet comes into existence. Class variables are not thread safe so
+     * should rarely be used and then only accessed in a synchronized block.
+     * Cannot communicate with the servlet container therefore init is preferred
+     * for initialization tasks. Generally not implemented.
+     */
+    public TheLearningServlet() {
+        LOG.info(">>> Constructor <<<");
+    }
+
+    /**
+     * The servlet initializer
+     *
+     * Called once the servlet is constructed. Can interact with the servlet
+     * container by calling getServletContext(). Preferred method to initialize
+     * anything that will be available to every thread.
+     *
+     * @throws javax.servlet.ServletException
+     */
+    @Override
+    public void init() throws ServletException {
+        LOG.info(">>> init <<<");
+    }
+
+    /**
+     * Destructor
+     *
+     * Java does not have the automatic destructor of C++ only the less than
+     * useful finalize(). This method is called by the server just before the
+     * servlet is removed from memory. Here you can release any resources that
+     * were available to all threads of the servlet that you likely created in
+     * init().
+     */
+    @Override
+    public void destroy() {
+        LOG.info(">>> destroy <<<");
+    }
+
+    /**
+     * getter for a String of information
+     *
+     * Returns information about the servlet, such as author, version, and
+     * copyright that you can add. Usually called by the app server similar to
+     * how the attribute 'description' is used in @WebServlet.
+     *
+     * @return
```

Comment:
I'd drop empty javadoc tags, as they provide no additional info, like this return above, or params/throws below.